packages/nc-gui/components/smartsheet/grid/Table.vue

```diff
@@ -1545,7 +1545,7 @@ const expandAndLooseFocus = (row: Row, col: Record<string, any>) => {
                 Clear
               </div>
             </a-menu-item>
-
+            <NcDivider class="!m-0" />
```

Comment:
Remove the margin override and use `NcMenu` and `NcMenuItem` which will take care of this